lib/lattice_observer/observed/claims.ex

```diff
@@ -0,0 +1,110 @@
+defmodule LatticeObserver.Observed.Claims do
+  require Logger
+  alias __MODULE__
+  alias LatticeObserver.Observed.{Lattice, Provider, Actor}
+
+  @enforce_keys [:sub]
```

Comment:
We probably want to support `iss` too right? I believe modules also have to have others when they sign with wash, like version and revision, but I'm unsure if wascap strictly enforces those